python/cudf/cudf/comm/serialize.py

```diff
@@ -7,8 +7,9 @@
 # Series/DataFrame/Index/Column/Buffer/etc
 serializable_classes = (
     cudf.DataFrame,
+    cudf.Index,
+    cudf.MultiIndex,
```

Comment:
So `cudf.MultiIndex` inherits from `Index`. Will the right thing will be done when dask tries to serialize/deserialize a MultiIndex?

Reply:
Good point. We could drop `cudf.MultiIndex`. Though there is no harm in adding it. Happy either way 🙂

Reply:
TL;DR Just for clarity, in either case Dask will do the same thing.

This is just a consequence of the functions we register for serialization and deserialization. With Dask serialization, we register (`cuda_serialize_cudf_object`, `dask_serialize_cudf_object`), which just call the `.serialize(...)` method of the object given. So this will be an `Index` or `MultiIndex` instance. Similarly in the `deserialize_cudf_object` we track [the type to deserialize]( https://github.com/rapidsai/cudf/blob/1d1d0d702c431bcd08d959e28486dab1e6e17b14/python/cudf/cudf/comm/serialize.py#L51 ). So can faithfully deserialize a `MultiIndex` object or `Index` object based on which was stored.

Reply:
Thanks for looking into it. I agree - let's keep it as is for now.